next/frontend/hooks/useServerSideAuth.ts

```diff
@@ -10,6 +10,12 @@ export const useServerSideAuth = () => {
     ...serverSideAuthContext,
     isAuthenticated: !!userData,
     accountType: userData?.['custom:account_type'],
+    // helper, since we usually determine what to display this way, slightly convoluted because of ts rules & undefined vs boolean
+    isLegalEntity: userData?.['custom:account_type']
+      ? [AccountType.FyzickaOsobaPodnikatel, AccountType.PravnickaOsoba].includes(
+          userData?.['custom:account_type'],
+        )
+      : false,
```

Comment:
probably more readable?
```suggestion
   isLegalEntity: (
      [AccountType.FyzickaOsobaPodnikatel, AccountType.PravnickaOsoba] as (string | undefined)[]
    ).includes(userData?.['custom:account_type'])
```